DeepSpeech.ipynb

```diff
@@ -106,7 +107,14 @@
     "do_fulltrace = bool(len(os.environ.get('ds_do_fulltrace', '')))\n",
     "\n",
     "if do_fulltrace:\n",
-    "    check_cupti()"
+    "    check_cupti()\n",
+    "\n",
+    "track_gpu_usage = bool(len(os.environ.get('ds_track_gpu_usage', '')))\n",
+    "gpu_usage_charts = os.environ.get('ds_gpu_usage_charts', 'gpu_usage_%%s_%d.png' % int(time.time()))\n",
+    "\n",
+    "if track_gpu_usage:\n",
+    "    gu = GPUUsage()\n",
+    "    gu.start()"
```

Comment:
Should be moved to after 
```python
data_sets.start_queue_threads(session)
```
so the threads start at a time TensorFlow deems is OK[[1](https://www.tensorflow.org/versions/r0.11/how_tos/reading_data/index.html#reading-from-files)].